src/oca_github_bot/tasks/tag_ready_to_merge.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) ACSONE SA/NV 2018
+# Copyright (c) https://github.com/madman-bob 2018
+# Distributed under the MIT License (http://opensource.org/licenses/MIT).
+
+from datetime import datetime, timedelta
+
+from .. import github
+from ..github import gh_call
+from ..queue import task, getLogger
+
+_logger = getLogger(__name__)
+
+
+APPROVALS_REQUIRED = 2
+MIN_PR_AGE = 5  # days
```

Comment:
Maybe we could use something like that to avoid counting down on Saturdays and Sundays by default.
https://gist.github.com/JosefJezek/275b2c667698aabd6969

Reply:
Yes, that would be a nice improvement. I'll try to make the basic version work first, then we can improve.

Reply:
@damdam-s if you think it's important to not count week-ends in the 5 days, can you open an issue for discussion?

Reply:
@sbidoul #17 added to discuss